### DISCUSSION TITLE
0 recognized as 3?

### DISCUSSION THREAD

**Discussion by odrobnik** [jomjol/AI-on-the-edge-device#3371] [General]
How do I need to configure it such that 0 is recognized not as 3?

<img width="1010" alt="image" src="https://github.com/user-attachments/assets/fa3a66d6-23b6-46c6-bf27-0ed6a2c68032">

**Comment by SybexX** [comment]
https://jomjol.github.io/AI-on-the-edge-device-docs/Choosing-the-Model/
